delocate/delocating.py

```diff
@@ -135,7 +168,16 @@ def copy_recurse(lib_path, copy_filt_func=None, copied_libs=None):
     copied_libs : dict
         Input `copied_libs` dict with any extra libraries and / or dependencies
         added.
+
+    .. deprecated:: 0.8
```

Comment:
Also - add note in `Changelog` ?

Reply:
Where do I add it?  May changelogs always have an 'unreleased' section for unreleased updates.